spec/models/wishlist_spec.rb

```diff
@@ -1,6 +1,13 @@
 require "rails_helper"
 
 RSpec.describe Wishlist, type: :model do
```

Comment:
This isn't my change - it's already on the master.
I did this in the other spec though to make it consistent.